actors/builtin/miner/monies.go

```diff
@@ -71,19 +75,19 @@ func PledgePenaltyForTermination(initialPledge abi.TokenAmount, sectorAge abi.Ch
 
 // Computes the PreCommit Deposit given sector qa weight and current network conditions.
 // PreCommit Deposit = 20 * BR(t)
-func PreCommitDepositForPower(epochTargetReward abi.TokenAmount, networkQAPower abi.StoragePower, qaSectorPower abi.StoragePower) abi.TokenAmount {
-	return big.Mul(PreCommitDepositFactor, ExpectedDayRewardForPower(epochTargetReward, networkQAPower, qaSectorPower))
+func PreCommitDepositForPower(rewardEstimate, networkQAPowerEstimate *smoothing.FilterEstimate, qaSectorPower abi.StoragePower) abi.TokenAmount {
+	return big.Mul(PreCommitDepositFactor, ExpectedDayRewardForPower(rewardEstimate, networkQAPowerEstimate, qaSectorPower))
 }
 
 // Computes the pledge requirement for committing new quality-adjusted power to the network, given the current
 // total power, total pledge commitment, epoch block reward, and circulating token supply.
 // In plain language, the pledge requirement is a multiple of the block reward expected to be earned by the
 // newly-committed power, holding the per-epoch block reward constant (though in reality it will change over time).
-func InitialPledgeForPower(qaPower abi.StoragePower, networkQAPower, baselinePower abi.StoragePower, networkTotalPledge abi.TokenAmount, epochTargetReward abi.TokenAmount, networkCirculatingSupply abi.TokenAmount) abi.TokenAmount {
+func InitialPledgeForPower(qaPower abi.StoragePower, baselinePower abi.StoragePower, networkTotalPledge abi.TokenAmount, rewardEstimate, networkQAPowerEstimate *smoothing.FilterEstimate, networkCirculatingSupplySmoothed abi.TokenAmount) abi.TokenAmount {
+	networkQAPower := networkQAPowerEstimate.Estimate()
+	ipBase := big.Mul(InitialPledgeFactor, ExpectedDayRewardForPower(rewardEstimate, networkQAPowerEstimate, qaPower))
```

Comment:
However, I'm not sure if what I had in mind is actually a good idea. It would certainly help reduce early pledge requirements, but would be a deviation from the idea that there is a single number `BR` which is used to compute both IP and fees (FF, SP), which is something that may be relied upon by Red Team analysis.

Reply:
One way around this would be to compute the "daily BR" number as an extrapolation of total reward over the course of 20 days, divided by 20. That way, IP is exactly the extrapolated reward over 20 days, and it also has the previously specified relationships to SP, FF, etc. I'd like to get input from @nicola and @zixuanzh on this.

Reply:
Great to know. I'll follow up once there is a concrete plan, likely after I merge this PR.  Tracked in #822

Reply:
I am not sure I understand the issue, if I am still needed feel free to ping me.